cp/gtpv2c_messages/delete_session.c

```diff
@@ -117,13 +117,18 @@ delete_context(delete_session_request_t *ds_req,
 			 */
 			si.bearer_id = ds_req->linked_ebi.eps_bearer_id;
 			si.ue_addr.u.ipv4_addr =
-				htonl(pdn->ipv4.s_addr);
+				(pdn->ipv4.s_addr);
 			si.ul_s1_info.sgw_teid =
 				htonl(bearer->s1u_sgw_gtpu_teid);
 			si.sess_id = SESS_ID(
 					context->s11_sgw_gtpc_teid,
 					si.bearer_id);
 			struct dp_id dp_id = { .id = context->dpId };
```

Comment:
some are htonl others are not. within same msg

Reply:
Can't do anything about this at the moment. `create_session` (and other messages) send `teids`, enodeB IPs in different byte orders. If I change other lines over here, I would end up introducing more scrambled message formats elsewhere.